buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -40,7 +40,9 @@ fun Project.createStackDeployTask(profile: String) {
                         // so we create it here only in dev profile
                         """|$it
                            |  mysql:
-                           |    image: mysql:8.0.20
+                           |    image: mysql
+                           |    platform: linux/x86_64
```

Comment:
could you please check that with latest (not 8.0.20) version of mysql EVEN without a platform flag everything works fine

Reply:
Latest version still doesn't work on M1 without `platform` flag, so the flag is needed.

Reply:
Works fine without `platform` but with `:8.0.28-oracle` tag.